### PR TITLE
Created Seychelles legislation preprocesing script

### LINKED ISSUE #12: Develop preprocessing script for legislation from Seychelles

### PR DESCRIPTION
**Pull request by Skyrolon**
Closes #12

### CODE REVIEW

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
... purposes of protecting the catchment areas used for abstracting water for supplying to the Community. [b] The Minister may by regulations prohibit or regulate in the protection zone activities which may affect the quality of the water directly or indirectly. [4][a] No person shall, without an authorisation from the Authority, discharge or place into the ground or dispose of in the sub-soil or dig into the ground any polluting or hazardous substance or waste. [b] An authorisation issued under paragraph [a] above may be modified or revoked at any time by the Authority, if there is a risk of pollution of the water-supply source or for any other consideration for the protection of the environment. [c] An owner or user of any land shall immediately notify to the Authority any incidents of pollution of the soil or sub-soil owned or used by such owner or user. [5] No person shall, without an authorisation from the Authority, discharge any effluent or throw, deposit, or place any polluting or hazardous substance or waste or any obstructing matter into any watercourse or in the territorial waters. [6] A person who contravenes subsection [4][a] or [5] is guilty of an offence under this Act.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `[2][a]` with `[2] [a]` (i.e. add a space between the two elements). Also, please replace `[3][a]` with `[3] [a]`. Furthermore, please replace the first instance of `[4][a]` with `[4] [a]`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+(9) [1] The Minister may, on the recommendations of the Authority, by regulations, establish noise emission standards and such noise abatement measures as may be necessary to preserve the environment. [2] No person shall emit noise in excess of the noise emission standards established under this Part, except where an authorisation has been granted by the Authority allowing excessive emission of noise under such terms and conditions as it may determine. [3] The terms and conditions of an authorisation under sub-section [2] shall include compliance with such measures required to adequately protect persons exposed to excessive levels of noise. [4] A person who contravenes subsection [2] is guilty of an offence under this Act.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `sub-section` with `subsection` (the latter is used 34 times in this legislation, while the former is only used four times).

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
...] an estimate of the quantities of sand, coral, sea shells and other substances being removed from the Coastal Zone; [h] an estimate of the impact of erosion on the Coastal Zone; [i] an estimate of the extent, nature, causes and sources of coastal pollution and degradation; [j] any other relevant data or information that may be deemed necessary. [4] No person shall release or cause to be released into the Coastal Zone such polluting or hazardous substances by dumping or through the atmosphere. [5][a] The Minister may make such regulations as deemed necessary to ensure that activities in the Coastal Zone are so conducted as not to cause damage to the environment and to provide for such measures as are necessary to prevent, reduce and control pollution in the Coastal Zone. [b] Notwithstanding the generality of paragraph [a], the regulations may provide for: [i] the control and prevention of pollution of the marine environment from land-based sources, including rivers, estuaries, pipelines and outfall structures; [ii] such other matters relating to the preservation and conservation of the Coastal Zone. [6] A person who contravenes subsection [4] is guilty of an offence under this Act.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `[2][a]` with `[2] [a]`. Also, please replace `[5][a]` with `[5] [a]`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
...section [4][b]. [6] No person shall dispose of any hazardous waste except in accordance with the authorisation from the Agency. [7] No person shall import any hazardous waste into Seychelles. [8][a] No person shall transport hazardous waste within or through Seychelles without an authorisation from the Agency. [b] No person shall export hazardous waste to any country without obtaining an authorisation from the Agency and the prior informed consent of the receiving country. [9] The Agency shall: [a] designate, monitor and regulate waste disposal sites; [b] monitor the contamination and degradation of the environment arising from the operation of any waste disposal site; [c] monitor the safety and health of workers at waste disposal sites; [d] maintain the statistical data on the nature, quantity and volume of waste generated, and on sites and waste processing, where waste disposal is taking or has taken place; [e] do all such things as appear to be reasonably necessary for the monitoring and control of waste. [10] A person who contravenes subsection [3] or subsection [4][a] or [b] or subsection [6] or subsection [7] or subsection [8][a] or [b] is guilty of an offence under this Act.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `bin: [ii]` with `bin; [ii]`. Also, please replace `[8][a]` with `[8] [a]`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+(14) [1] No person shall handle or cause to be handled any hazardous substance except in accordance with such procedures and after complying with such safeguards as may be prescribed. [2] "Handling", in relation to any substance, means the manufacture, processing, treatment, package, storage, transportation, importation, use, collection, destruction, conversion, offering for sale, transfer or the like of a substance. [3] A person who contravenes subsection [1] is guilty of an offence under this Act. 
+
+PART: IV ENVIRONMENT IMPACT ASSESSMENT
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `PART: IV ENVIRONMENT` with `PART IV - ENVIRONMENT`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
...which the implementation of the project depends. [9] Wherever project implementation as referred to in this part brings about a change in the physical environment, the project proponent shall restore the matters specified by the Authority. [10] Where the project proponent fails to do so the Authority may itself perform the work and recover the cost from the project proponent who, in any event, shall compensate for any resulting loss or damage to the environment. [11] The Minister may, with the approval of the Cabinet, in exceptional circumstances and by notification giving grounds on which the decision is based, exclude a prescribed project from the Environment Impact Assessment process. [12] The notification under subsection [11] shall be made public and shall state the measures which are deemed to be necessary in order to reduce the environmental impact of the activity or the project. [13] Any person aggrieved by the decision or order of the Competent Authority may appeal to the Minister in the prescribed manner, and the Minister may affirm, revoke or vary the decision or order. [14] A person who fails to comply with a direction given under subsection [7] is guilty of an offence.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `[4][a]` with `[4] [a]`. Also, please replace `[5][a]` with `[5] [a]`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+PART V - ENFORCEMENT
+
+(16) [1] Where the Administrator is of the opinion that a person is contravening, or is likely to contravene any provision of this Act, the Administrator may cause to be served on the person an enforcement notice: [a] stating that the Administrator is of the said opinion; [b] specifying the matter constituting the contravention or the matters making it likely that the contravention will arise, as the case may be; [c] specifying the measures that shall be taken to remedy the contravention or eliminate the likelihood of a contravention as the case may be and [d] specifying the period within which the measures shall be implemented. [2] Any person who fails to comply with an enforcement notice under subsection [1] is guilty of an offence.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `case may be and [d]` with `case may be; and [d]`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+(23) The Minister may declare an environmental emergency: [a] where pollution occurs or is apprehended to occur due to any accident or other unforeseen act or event; or [b] where in the opinion of the Minister there is an imminent risk of serious pollution of the environment. in case of an environmental emergency declared under section 23, an authorised officer may, without warrant, arrest a person reasonably suspected of the contravention of this Act.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `environment. in` with `environment. In`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+(30) [1] Any person who commits an offence under section 16[2] or section 17[5] or section 22[2] or section 22[3] is liable on conviction, to imprisonment for one year and a fine of R50.000 and, if the offence is continued after conviction, is liable to a further fine of R5.000 for each day during which the offence is so continued. [2] A person who commits an offence under sections 7[4][a] or section 7[5] or section 8[2] or section 8[3] or section 9[2] or section 11[4] or section 12[3] or section 12[6] section 12[7] or section 12[8][a] or section 12[8][b] or section 14[1] or section 15[1] or section 15[14] is liable, on conviction, to imprisonment for 6 years and a fine of R250.000 and, if the offence is continued after conviction, is liable to a further fine of R5.000 for each day during which the offence is so continued. [3] Where an offence specified in subsection [2] is continued beyond a period of one year after conviction by the person so convicted the person is guilty of an offence and liable on conviction to imprisonment for not less than 2 years and not exceeding 7 years.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `sections 7[4][a]` with `section 7[4][a]`. Also, please replace `12[6] section` with `12[6] or section`. Furthermore, please replace `50.000`, `5.000`, `250.000` and `5.000` with `50,000`, `5,000`, `250,000` and `5,000` respectively.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
...amount of fixed penalty; and [c] the person to whom and the address at which the fixed penalty may be made or paid. [5] A fixed penalty specified in a notice under subsection [4] may be paid in cash to the person specified in the notice or by cheque enclosed in a letter addressed to that person at the address specified in the notice. [6] Where a letter is sent in accordance with subsection [5], the payment shall be regarded as having been made at the time at which that letter would be delivered in the ordinary course of post. [7] The form of notices under this section shall be such as may be prescribed under the regulations. [8] The fixed penalty payable in pursuance of a notice under this section shall, subject to subsection [9], be R100. [9] The Minister may by order substitute a different amount for the time being specified as the amount of the fixed penalty in subsection [8]. [10] In any proceedings a certificate which purports to be signed by or on behalf of the person receiving the fixed penalty stating that the payment of a fixed penalty was or was not received by a date specified in the certificate, shall, without proof of signature, be evidence of the facts stated therein.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `5.000` with `5,000`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+
+(31) Any person who: [a] destroys , pulls down, removes, injures or defaces any pillar, post or stake fixed by the Authority or any notice or other matter put up, inscribed or placed by or under the order of the Authority; or [b] obstructs any person acting under the orders or directions of the Authority from exercising the powers and performing the functions under this Act; or [c] damages any works or property belonging to the Authority, or [d] fails to furnish to any officer or other employee of the Authority a report or any information required by the officer or employee for the purpose of this Act, or [e] fails to intimate the occurrence of any accident or other unforeseen act to the Authority, or [f] in giving a report or any information which the person is required to give under this Act, makes a statement which is false or misleading in any material particular, is guilty of an offence and is liable on conviction to imprisonment for 6 months and a fine of R10.000.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace:
* `destroys , pulls` with `destroys, pulls`;
* `Authority, or [d]` with `Authority; or [d]`;
* `Act, or [e]` with `Act; or [e]`;
* `Authority, or [f]` with `Authority; or [f]`;
* `particular, is` with `particular; is`; and
* `10.000` with `10,000`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
...any offence under this Act has been committed by a company, every person who, at the time of offence was committed, was in charge of, and was responsible to, the company for the conduct of the business of the company, as well as the company is guilty of the offence and liable to be proceeded against and punished accordingly: Provided that nothing contained in this sub-section shall render any such person liable to any punishment provided in this Act, if the person proves that the offence was committed without the knowledge of that person or that the person exercised all due diligence to prevent the commission of such offence. [2] Where an offence under this Act has been committed with the consent or connivance of, or is attributable to any neglect on the part of any director, manager, secretary or other officer of the company, such director, manager, secretary or other officer is guilty of that offence and shall be liable to be proceeded against and punished accordingly. [3] For the purposes of this section: [a] "company" means any body corporate and includes a partnership or other association of individuals; [b] "director" in relation to a partnership, means a partner in the firm.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `the time of offence was committed` with `the time the offence was committed`. Also, please replace `accordingly: Provided` with `accordingly, provided`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+
+PART VII - MISCELLANEOUS PROVISIONS
+
+(35) The Minister may, by notification in the official gazette, delegate, subject to such conditions and limitations as may be specified in the notification, such of the powers and functions under this Act (except the power to make regulations) as the Minister may deem necessary or expedient, to any person or public authority.
+
+(36) [1] Where the discharge of any pollutant in excess of the prescribed standards occurs or is apprehended to occur due to any accident or other unforeseen act or event, the person responsible for such discharge and the person in charge of the premises shall be bound to prevent or mitigate the pollution caused as a result of such discharge and shall also forthwith: [a] intimate the fact of such occurrence or apprehension of such occurrence; and [b] be bound, if called upon, to render all assistance, to the Authority or such other authorities or agencies as may be prescribed. [2] A person who fails to comply with subsection [1] is guilty of an offence and is liable on conviction to a fine of R10.000 and to imprisonment for one year.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `10.000` with `10,000`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+
+(37) [1] The Authority may, in relation to its functions under this Act, from time to time, require any person, officer or other authority to furnish to it or to any prescribed authority or officer any reports, returns, statistics, accounts and other information that may be required for the purposes of this Act and such person , officer, or other authority shall be bound to do so. [2] A person who fails to comply with subsection [1], on being required by the Authority to do so, is guilty of an offence and is liable on conviction to a fine of R10.000.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `person ,` with `person,`. Also, please replace `10.000` with `10,000`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 15]
```diff
@@ -0,0 +1,91 @@
...
+
+(38) Where any officer of the Authority or any person appointed on a committee or any person discharging any function or duty under this Act discloses otherwise than in the performance of the duty of the officer or such person any information relating to any trade secret used in carrying on a particular project or activity and the information has been given to the officer or such person by virtue of this Act, the officer or such person is guilty of an offence and is liable on conviction, to a fine of R10.000 and to imprisonment for one year.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `10.000` with `10,000`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 16]
```diff
@@ -0,0 +1,91 @@
...
+
+(39) No suit or prosecution or other legal proceedings shall lie against the Minister or an officer of the Authority or an officer empowered under this Act, for anything that is done or intended to be done in good faith in pursuance of this Act or the regulations made thereunder.
+
+(40) [1] The Minister may make regulations under this Act for all such matters as are to be prescribed under the Act and for any other matter that may be deemed necessary in pursuance of the objects of this Act, for the protection of any aspect of the environment and for the prevention, control and abatement of environmental pollution. [2] Regulations made under this section may provide for offences and penalties not exceeding R20.000 by way of fine and 2 years by way of imprisonment in respect of those offences.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `20.000` with `20,000`.

**Review by danpdraper — CHANGES_REQUESTED** [review]

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 17]
```diff
@@ -0,0 +1,91 @@
...
+
+(14) [1] No person shall handle or cause to be handled any hazardous substance except in accordance with such procedures and after complying with such safeguards as may be prescribed. [2] "Handling", in relation to any substance, means the manufacture, processing, treatment, package, storage, transportation, importation, use, collection, destruction, conversion, offering for sale, transfer or the like of a substance. [3] A person who contravenes subsection [1] is guilty of an offence under this Act. 
+
+PART - IV ENVIRONMENT IMPACT ASSESSMENT
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `PART - IV` with `PART IV -`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+
+(30) [1] Any person who commits an offence under section 16[2] or section 17[5] or section 22[2] or section 22[3] is liable on conviction, to imprisonment for one year and a fine of R50,000 and, if the offence is continued after conviction, is liable to a further fine of R,000 for each day during which the offence is so continued. [2] A person who commits an offence under section 7[4][a] or section 7[5] or section 8[2] or section 8[3] or section 9[2] or section 11[4] or section 12[3] or section 12[6] or section 12[7] or section 12[8][a] or section 12[8][b] or section 14[1] or section 15[1] or section 15[14] is liable, on conviction, to imprisonment for 6 years and a fine of R250,000 and, if the offence is continued after conviction, is liable to a further fine of R,000 for each day during which the offence is so continued. [3] Where an offence specified in subsection [2] is continued beyond a period of one year after conviction by the person so convicted the person is guilty of an offence and liable on conviction to imprisonment for not less than 2 years and not exceeding 7 years.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace the two instances of `R,000` with `R5,000`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 19]
```diff
@@ -0,0 +1,91 @@
...
...amount of fixed penalty; and [c] the person to whom and the address at which the fixed penalty may be made or paid. [5] A fixed penalty specified in a notice under subsection [4] may be paid in cash to the person specified in the notice or by cheque enclosed in a letter addressed to that person at the address specified in the notice. [6] Where a letter is sent in accordance with subsection [5], the payment shall be regarded as having been made at the time at which that letter would be delivered in the ordinary course of post. [7] The form of notices under this section shall be such as may be prescribed under the regulations. [8] The fixed penalty payable in pursuance of a notice under this section shall, subject to subsection [9], be R100. [9] The Minister may by order substitute a different amount for the time being specified as the amount of the fixed penalty in subsection [8]. [10] In any proceedings a certificate which purports to be signed by or on behalf of the person receiving the fixed penalty stating that the payment of a fixed penalty was or was not received by a date specified in the certificate, shall, without proof of signature, be evidence of the facts stated therein.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Please replace `Rs. [,000][2]` with `R5,000. [2]`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 20]
```diff
@@ -0,0 +1,91 @@
...
... any offence under this Act has been committed by a company, every person who, at the time of offence was committed, was in charge of, and was responsible to, the company for the conduct of the business of the company, as well as the company is guilty of the offence and liable to be proceeded against and punished accordingly: Provided that nothing contained in this subsection shall render any such person liable to any punishment provided in this Act, if the person proves that the offence was committed without the knowledge of that person or that the person exercised all due diligence to prevent the commission of such offence. [2] Where an offence under this Act has been committed with the consent or connivance of, or is attributable to any neglect on the part of any director, manager, secretary or other officer of the company, such director, manager, secretary or other officer is guilty of that offence and shall be liable to be proceeded against and punished accordingly. [3] For the purposes of this section: [a] "company" means any body corporate and includes a partnership or other association of individuals; [b] "director" in relation to a partnership, means a partner in the firm.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: This article still needs the following two amendments:
* `the time of offence was committed` for `the time the offence was committed`; and
* `accordingly: Provided` for `accordingly, provided`.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 21]
```diff
@@ -0,0 +1,91 @@
...
+
+(37) [1] The Authority may, in relation to its functions under this Act, from time to time, require any person, officer or other authority to furnish to it or to any prescribed authority or officer any reports, returns, statistics, accounts and other information that may be required for the purposes of this Act and such person , officer, or other authority shall be bound to do so. [2] A person who fails to comply with subsection [1], on being required by the Authority to do so, is guilty of an offence and is liable on conviction to a fine of R10,000.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: `person ,` still needs to be replaced with `person,`.

**Review by danpdraper — CHANGES_REQUESTED** [review]

File: raw_data/preprocessed/africa/seychelles_english.txt (hunk 20 above)
Comment (Skyrolon) [reply raw_data/preprocessed/africa/seychelles_english.txt]: Not sure why these amendments didn't go through as they were in the code, but they are there now.

File: raw_data/preprocessed/africa/seychelles_english.txt [hunk 22]
```diff
@@ -0,0 +1,91 @@
...
...amount of fixed penalty; and [c] the person to whom and the address at which the fixed penalty may be made or paid. [5] A fixed penalty specified in a notice under subsection [4] may be paid in cash to the person specified in the notice or by cheque enclosed in a letter addressed to that person at the address specified in the notice. [6] Where a letter is sent in accordance with subsection [5], the payment shall be regarded as having been made at the time at which that letter would be delivered in the ordinary course of post. [7] The form of notices under this section shall be such as may be prescribed under the regulations. [8] The fixed penalty payable in pursuance of a notice under this section shall, subject to subsection [9], be R100. [9] The Minister may by order substitute a different amount for the time being specified as the amount of the fixed penalty in subsection [8]. [10] In any proceedings a certificate which purports to be signed by or on behalf of the person receiving the fixed penalty stating that the payment of a fixed penalty was or was not received by a date specified in the certificate, shall, without proof of signature, be evidence of the facts stated therein.
```
Comment (danpdraper) [inline raw_data/preprocessed/africa/seychelles_english.txt]: Final edit: `5.000` still needs to be replaced with `5,000`.

**Review by danpdraper — CHANGES_REQUESTED** [review]

File: raw_data/preprocessed/africa/seychelles_english.txt (hunk 22 above)
Comment (Skyrolon) [reply raw_data/preprocessed/africa/seychelles_english.txt]: Sorry about that, missed that one.

**Pull request merged**